libdnf/module/module_item.cpp

```diff
@@ -122,6 +122,17 @@ std::string ModuleItem::get_name_stream_context() const {
 }
 
 
+std::string ModuleItem::get_name_stream_staticcontext_arch() const {
+    return libdnf::utils::sformat(
```

Comment:
We should prefer to use `fmt::format` over `libdnf::utils::format`. Since C++20, `fmt::format` does the syntax checking of the format string during compilation. -> And it requires the format string to be a constant expression.
We have `libdnf::utils::sformat` for cases where the format string is not a constant expression, usually when the content of the format string are unknown at compile time.  For example, the format string is the result of translating into the user's language (localization).

Reply:
So, in this case, `fmt::format` can't be used? Because computed_static_context is not known at compile time?

Reply:
In this case we should use `fmt::format`. The format string "{}:{}:{}:{}" is known at compile time.

Example of incorrect code - The format string requires more arguments than are passed.:
- `fmt::format("{}:{}:{}:{}", "a", "b", "c")` reports an error at compile time.  Which can be used by the IDE.
- `libdnf::utils::sformat("{}:{}:{}:{}", "a", "b", "c")` parses the format string at runtime. We get the exception at runtime.

Reply:
I see. Ok, I replaced the `libdnf::utils::sformat` with `fmt::format` where possible in the last commit.